src/diffbot.js

```diff
@@ -1,9 +1,13 @@
+const axios = require('axios');
```

Comment:
Pulling in a dependency for Axios is overkill for a simple library. I'd rather keep the dependencies down and just use fetch.